APickup/APickup.h

```diff
@@ -63,6 +66,8 @@ class VRBASE_API APickup : public ABaseEntity {
         Super::PostEditChangeProperty(PropertyChangedEvent);
     }
 #endif
+    UFUNCTION(BlueprintCallable)

```

Comment:
Why is this `BlueprintCallable`? Why not `BlueprintNativeEvent`?

Reply:
The keyword `BlueprintCallable` allows a function to be called by a blueprint in the Unreal editor. The keyword `BlueprintNativeEvent` is used when it is expected or planned for the function to be overridden in the editor.
Documentation is here:
https://docs.unrealengine.com/en-US/Programming/UnrealArchitecture/Reference/Functions/index.html

There is no reason for the function to be overridden in the editor, so `BlueprinCallable was implemented.

Reply:
Seen 3:01 PM